audiorecorder

```diff
@@ -330,7 +330,7 @@ if [ -e AUDIORECORDERTEMP.wav ] ; then
 fi
 
 #ASTATS_OUT=$(mktemp /tmp/temp.XXXX)
-rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 2000000 -p remix 1 2 | ffmpeg -i - -f wav -c:a "${CODEC}" -ar "${SAMPLE_RATE_NUMERIC}" -metadata comment="" -filter_complex "${CHANNELS}" -map "${LAYOUT}" -y -rf64 auto AUDIORECORDERTEMP.wav -f wav -c:a pcm_s16le -ar 44100 -filter_complex "${CHANNELS}" -map "${LAYOUT}" - |\
+rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 2000000 -p remix ${SOXCHANNELS} | ffmpeg ${CHANNEL_LAYOUT} -i - -f wav -c:a "${CODEC}" -ar "${SAMPLE_RATE_NUMERIC}" -metadata comment="" -y -rf64 auto AUDIORECORDERTEMP.wav -f wav -c:a pcm_s16le -ar 44100 - |\
 ffplay -window_title "Currently Recording Channel(s): ${channel_layout}" -f lavfi \
 "amovie='pipe\:0',${FILTER_CHAIN}" 2> /dev/null
```

Comment:
Honestly not sure why I broke that line up in the first place. I think I'll just bring it up! (especially since it now isn't even in the middle of the pipe sequence).